src/amo/components/Addon/styles.scss

```diff
@@ -148,6 +148,46 @@
   order: 4;
 }
 
+.AddonDescription-more-addons {
+  margin-top: 0;
+
+  .Card-contents .AddonsCard-list {
+    background-color: $white;
+    display: grid;
+    grid-auto-flow: initial;
+    grid-template-columns: repeat(2, 1fr);
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.AddonDescription-more-addons--theme {
+  .Card-contents .AddonsCard-list {
+    .SearchResult-result {
+      width: 100%;
+    }
+
+    .SearchResult-icon-wrapper {
+      margin: 0 0 10px;
+      width: 100%;
+    }
+
+    .SearchResult-icon {
+      border-radius: $border-radius-default;
+    }
+
+    @include respond-to(large) {
+      .SearchResult.SearchResult--theme {
+        margin: 0 5px;
+      }
+    }
+
+    @include respond-to(desktop) {
```

Comment:
We don't (or shouldn't) have a size named `desktop` referring to a device. @muffinresearch and I chatted about this ages ago so I can't find the thread but we decided we'd stick to using generic "size" descriptors. I think right now we have `medium`, `large`, and `extraLarge`.

Reply:
Also looking at the screenshots seems like this would work on `extraLarge`. But maybe I'm wrong–if we need the new size let's just relabel it.